lib/croods/controller/member.rb

```diff
@@ -23,14 +23,16 @@ def identifier
         params[resource.identifier]
       end
 
-      def member_params
+      def member_params # rubocop:disable Metrics/AbcSize
```

Comment:
how about we disable this for the project?
these ad-hoc rubocop disable reduce legibility and don't help with keeping the code standards (one of rubocop main purposes).